test/helpers/types/types.go

```diff
@@ -272,11 +277,26 @@ type AllLog struct {
 	Annotations              Annotations      `json:"annotations"`
 	K8SAuditLevel            string           `json:"k8s_audit_level"`
 	OpenshiftAuditLevel      string           `json:"openshift_audit_level"`
+	OpenshiftLabels          OpenshiftMeta    `json:"openshift"`
+}
+
+func StrictlyParseLogs(in string, logs interface{}) error {
+	logger.V(3).Info("ParseLogs", "content", in)
```

Comment:
i will suggest not to print the logs here. sometimes the volume of logs can be quite high, and it fills up the CI console. Then it becomes difficult to find the error from CI logs.

suggest to print the logs only if the parsing fails. OR just print something like "logs parsing failed, error=%v. Please look in the collected test artifacts"

Reply:
@vimalk78 I thought about that but decided that this is not creating so many logs ... on the flip side this is really good way to understand how things are working and how the test is comparing existing with required fields ... so decided to keep that in (for now) ... I am planning to add in the next PR some more tests (audit etc...) that use this functionality and want to follow up on the process. Once we will feel more confident that this is really comparing well and doing it's work ... totally agree we can remove this print message.